Ginger/Ginger/ApplicationModelsLib/OldPOMsLib/AddEditPOMWizardLib/UnmappedElementsPage.xaml.cs

```diff
@@ -62,6 +62,7 @@ private void AddButtonClicked(object sender, RoutedEventArgs e)
             foreach (ElementInfo EI in ItemsToAddList)
             {
                 EI.Selected = false;
+                EI.IsAutoLearned = false;
                 mPOM.MappedUIElements.Add(EI);
```

Comment:
This is not looks like correct place because "UnmappedElementsPage" not supposed to be used.

Reply:
Removed